src/pyspiffe/bundle/jwt_bundle/jwt_bundle_set.py

```diff
@@ -1,28 +1,43 @@
+"""
+This module manage JwtBundleSet objects.
+"""
+
 from typing import Mapping
 from pyspiffe.bundle.jwt_bundle.jwt_bundle import JwtBundle
 from pyspiffe.spiffe_id.trust_domain import TrustDomain
 
 
 class JwtBundleSet(object):
-    bundles: Mapping[TrustDomain, JwtBundle]
+    """JwtBundleSet is a set of JWTBundles objects, keyed by trust domain."""
 
-    def new_set(
-        self, bundles: Mapping[TrustDomain, JwtBundle]
-    ) -> Mapping[TrustDomain, JwtBundle]:
-        """
-        NewSet creates a new, empty set.
-        """
-        raise Exception('not implemented')
+    def __init__(self, bundles: Mapping[TrustDomain, JwtBundle]) -> None:
+        """Creates a new initialized with the given JWT bundles.
+
+        Args:
+            bundles: a set of JwtBundles to initialize the JwtBundleSet
```

Comment:
nit: after semicolom, use capital letter. So, it should be:
`bundles: A set of Jwt`
Please, review the other files for this one.